Translations/fr.po

```diff
@@ -7806,9 +7810,9 @@ msgid ""
 "need something to use against this King of the undead, then I can help you "
 "out..."
 msgstr ""
-"Écoutez, je dirige une affaire ici. Je ne vends pas d'informations, et je me "
-"fiche d'un roi mort depuis plus longtemps que je ne vis. Si vous avez besoin "
-"de quelque chose à utiliser contre ce roi des morts-vivants, alors je peux "
+"Dites, j'ai un bizness ici. Je ne vends pas d'informations, et je me "
```

Comment:
"bizness" look like an "anachronisme" here no ? And a Anglicisme

Reply:
```suggestion
"Dites, j'ai une petite affaire ici. Je ne vends pas d'informations, et je me "
```

Reply:
I tried to reflect some informal language from a not quite frequentable boy. The setting is medieval fantastic, but as such unrelated to the real world (people don’t have to speak Old French), so I don’t consider the word "bizness" is out of its time here. Anglicism or not, this word is meant to illustrate "some fishy, barely legal business", as we can see in some suburban areas in the real world. "Petite affaire" may suit, but lacks a little semantic strength in my opinion.